packages/web-components/@sfx/autocomplete/test/unit/autocomplete.test.ts

```diff
@@ -1,11 +1,12 @@
-import { expect, spy, stub } from '../utils';
-import { Base } from '@sfx/base';
 import { TemplateResult } from 'lit-element';
+import { Base } from '@sfx/base';
```

Comment:
We seem to be going back and forth on how to organize imports. Are we going with alphabetical (this) or the proposed separation of local and third party imports? 

This goes for the other imports changed in this PR as well.

Reply:
For tests, I've been saying: non-local, test utils, class under test, supporting local imports. For this particular one, I think this is correct. Here's an [example from another PR](https://github.com/groupby/sfx-view/pull/15/files/963ff79939af50f82d9d562e37672788bd193d71#discussion_r311100138).